bot/seasons/evergreen/movie.py

```diff
@@ -0,0 +1,225 @@
+                    movie_text += f"**Revenue:** ${movie_data['revenue'] if movie_data['revenue'] != 0 else '?'}\n"
+                    movie_text += f"**Duration:** {movie_data['runtime']} minutes\n\n"
+
+                    # Add description
+                    movie_text += movie_data['overview']
+
+                    # Define Movie Image URL
+                    movie_img_url = f"http://image.tmdb.org/t/p/w200{movie_data['poster_path']}"
+
+                    # Append this page to pages
+                    pages.append((movie_text, movie_img_url))
+        except KeyError as err:
+            # Create error message
+            msg = f"There was KeyError while executing HTTP request. API may " \
+                f"down or API key may be incorrect, however, some movies " \
+                f"have some missing fields, and this error will raise this " \
+                f"too. Problematic Key: \n```{err}``` "
+
+            # Create error embed
+            err_embed = Embed(title=":no_entry: Error :no_entry:")
+
+            # Log error
+            logger.warning(msg)
+
+            # Return error message + embed
+            return [(msg, "")], err_embed
+
+    # Return all movies pages
+    return pages, embed
+
+
+class Movie(Cog):
+    """Movie Cog contains movies command that grab random movies from TMDB."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+        self.http_session: ClientSession = bot.http_session
+
+    @command(name='movies', aliases=['movie'])
+    async def movies(self, ctx: Context, genre: str = "", amount: int = 5) -> None:
+        """
+        Get random movies by specifing genre.
+
+        Also support amount parameter,
+        that define how much movies will be shown. Default 5
+
+        Available genres:
+        - Action
+        - Adventure
+        - Animation
+        - Comedy
+        - Crime
+        - Documentary
+        - Drama
+        - Family
+        - Fantasy
+        - History
+        - Horror
+        - Music
+        - Mystery
+        - Romance
+        - Science
+        - Thriller
+        - Western
+        """
+        # Check is there more than 20 movies specified, due TMDB return 20 movies
+        # per page, so this is max. Also you can't get less movies than 1, just logic
+        if amount > 20:
+            await ctx.send('You can\'t get more than 20 movies at once. (TMDB limits)')
+            return
+        elif amount < 1:
+            await ctx.send('You can\'t get less than 1 movies. Just logic.')
+            return
+
+        # Capitalize genre for getting data from Enum
+        genre = genre.capitalize()
+
+        # If invalid genre, send help message
+        if genre not in MovieGenres.__members__:
+            await ctx.send_help('movies')
+            return
+
+        # Get pages and embed of movies
+        pages, embed = await get_random_movies(self.http_session, amount, MovieGenres[genre].value, genre)
```

Comment:
An EAFP here with a try except can be clearer, you can change this part to this
```py
try:
    # Get pages and embed of movies
    pages, embed = await get_random_movies(self.http_session, amount, MovieGenres[genre].value, genre)
except KeyError:
    # If invalid genre, send help message
    await ctx.send_help('movies')
    return
```
That way, you do not need to access `MovieGenres.__members__` and do a membership check